reactos/drivers/usb/usbhub/fdo.c

```diff
@@ -2009,11 +2063,13 @@ USBHUB_FdoHandlePnp(
         case IRP_MN_QUERY_REMOVE_DEVICE:
         case IRP_MN_QUERY_STOP_DEVICE:
         {
+            DPRINT("IRP_MN_QUERY_STOP_DEVICE\n");
             Irp->IoStatus.Status = STATUS_SUCCESS;
             return ForwardIrpAndForget(DeviceObject, Irp);
         }
         case IRP_MN_REMOVE_DEVICE:
         {
+            DPRINT("IRP_MN_REMOVE_DEVICE\n");
             Irp->IoStatus.Status = STATUS_SUCCESS;
             IoCompleteRequest(Irp, IO_NO_INCREMENT);
```

Comment:
Because for roothub pdo on IRP_MN_QUERY_CAPABILITIES our libusb's hub_controller responds with " DeviceCapabilities->Removable = FALSE; " so DV will not test I think ;)